moveit_setup_assistant/templates/moveit_config_pkg_template/package.xml.template

```diff
@@ -20,6 +20,7 @@
   <run_depend>joint_state_publisher</run_depend>
   <run_depend>robot_state_publisher</run_depend>
   <run_depend>xacro</run_depend>
+  <run_depend>warehouse_ros_mongo</run_depend>
```

Comment:
Yes but for now they aren't that easy to install 😏

> On 19 Aug 2016, at 17:48, Dave Coleman <notifications@github.com> wrote:
> 
> In moveit_setup_assistant/templates/moveit_config_pkg_template/package.xml.template:
> 
> > @@ -20,6 +20,7 @@
> >    <run_depend>joint_state_publisher</run_depend>
> >    <run_depend>robot_state_publisher</run_depend>
> >    <run_depend>xacro</run_depend>
> > +  <run_depend>warehouse_ros_mongo</run_depend>
> All moveit config packages have depended on mongo in the past, so continuing that isn't a big deal assuming the mongo dependencies are easy to install and well maintained
> …
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

-- 
*We're hiring Software Developers! - visit our website for more details.*
------------------------------
*Shadow Robot Company Ltd.*
251 Liverpool Road, N1 1LX, UK
Registered Number 3308007 (England & Wales)
------------------------------